src/events/eventActions.js

```diff
@@ -60,5 +65,10 @@ export const startEventPolling = (queueId: number, eventId: number) => async (
     } else if (actions.length === 1) {
       dispatch(actions[0]);
     }
+
+    if (startTypingNotificationLoop) {
+      // first typing event after a while so start clearTypingNotification loop
+      dispatch(clearTypingNotification());
```

Comment:
This whole file should be more generic and should not have knowledge about any specific event type.